pkg/services/accesscontrol/resourcepermissions/api_test.go

```diff
@@ -448,9 +520,10 @@ var testOptions = Options{
 	Resource:          "dashboards",
 	ResourceAttribute: "id",
 	Assignments: Assignments{
-		Users:        true,
-		Teams:        true,
-		BuiltInRoles: true,
+		Users:           true,
+		Teams:           true,
+		BuiltInRoles:    true,
+		ServiceAccounts: true,
```

Comment:
had to add `ServiceAccounts: true` to assignments, probably can refactor this later